azure/public_ip.sh

```diff
@@ -0,0 +1,24 @@
+# Public IP related functions
+
+fn azure_public_ip_create(name, group, location, allocation) {
+       (
+               az network public-ip create --name $name --resource-group $group --location $location --allocation-method $allocation
```

Comment:
I know that `klb` still aborts in several places, but as discussed before I think we must be careful now and check exit status of new things added. Everything that can fail, should return the error (not abort).
`Klb` users can chose if they will do something regarding the failure or just ignore... 
What do you think?